packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -64,12 +73,21 @@ export class CfnInclude extends core.CfnElement {
   constructor(scope: core.Construct, id: string, props: CfnIncludeProps) {
     super(scope, id);
 
+    this.parameterValues = props.parameterValues;
+
     // read the template into a JS object
     this.template = futils.readYamlSync(props.templateFile);
 
     // ToDo implement preserveLogicalIds=false
     this.preserveLogicalIds = true;
 
+    // check if all user specified parameter values exist in the template
+    for (const logicalId of Object.keys(props.parameterValues || {})) {
+      if (!(logicalId in this.template.Parameters)) {
```

Comment:
Good catch... it does, but only if the user has specified a parameter to be replaced. This will throw an exception, but I'll update it so a better error message is thrown.